jupyterhub/auth.py

```diff
@@ -123,17 +127,32 @@ class LocalAuthenticator(Authenticator):
         should I try to create the system user?
         """
     )
-    system_user_home = Unicode('', config=True,
-        help="""Specify root home directory for users if different from system default.
+    add_user_cmd = Command(config=True,
+        help="""add user command
         
-        Passed to `useradd -b`.
-        If specified, when system users are created their home directories will be created in
+        USERNAME will be replaced with the user's name.
+        The user's name will be used as the last argument, e.g.
         
-            system_user_home/username
+        You may want to consult the difference between adduser and useradd.
         
-        Only has an effect when users are created with `create_system_users=True`.
+        useradd -d /home/USERNAME -s /bin/bash
```

Comment:
Also:

For Linux, the default value is `['adduser', '-q', '--gecos', '', '--disable-password']`. To specify a custom home directory, set this to  `['adduser', '-q', '--gecos', '', '--home', '/customhome/USERNAME', '--disable-password']`